gui-apps/hyprshot/hyprshot-1.2.3.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 2024 Gentoo Authors
+
```

Comment:
Missing license comment. It was present in the earlier revision.

e.g. `# Distributed under the terms of the GNU General Public License v2`